zwave_js_server/const.py

```diff
@@ -128,3 +129,53 @@ class CommandClass(IntEnum):
     ZIP_ND = 88
     ZIP_PORTAL = 97
     ZWAVEPLUS_INFO = 94
+
+
+class ThermostatMode(IntEnum):
+    """Enum with all (known/used) Z-Wave ThermostatModes."""
+
+    # https://github.com/zwave-js/node-zwave-js/blob/master/packages/zwave-js/src/lib/commandclass/ThermostatModeCC.ts#L53-L70
+    OFF = 0
+    HEAT = 1
+    COOL = 2
+    AUTO = 3
+    AUXILIARY = 4
+    RESUME_ON = 5
+    FAN = 6
+    FURNANCE = 7
+    DRY = 8
+    MOIST = 9
+    AUTO_CHANGE_OVER = 10
+    HEATING_ECON = 11
+    COOLING_ECON = 12
+    AWAY = 13
+    FULL_POWER = 15
+    MANUFACTURER_SPECIFIC = 31
+
+
+# In Z-Wave the modes and presets are both in ThermostatMode.
+# This list contains thermostatmodes we should consider a mode only
+MODES_LIST = [
+    ThermostatMode.OFF,
+    ThermostatMode.HEAT,
+    ThermostatMode.COOL,
+    ThermostatMode.AUTO,
+    ThermostatMode.AUTO_CHANGE_OVER,
+]
+
+# https://github.com/zwave-js/node-zwave-js/blob/master/packages/zwave-js/src/lib/commandclass/ThermostatSetpointCC.ts#L53-L66
+MODE_SETPOINT_MAP: Dict[int, List[str]] = {
```

Comment:
this one too. It is specific for thermostat CC so it should be in the constant's name imo.

Reply:
BTW: Isn't this also mapping HA modes to ZW modes ? In that case it belongs to the integration.
@MartinHjelmare what do you think ?

Reply:
IMO it is not - it's mapping enum values (Z-Wave protocol specific) to the values that are needed for ZWave JS (the string versions)

Reply:
yeah, you're right.